helpers/env-variables.ts

```diff
@@ -142,12 +144,12 @@ const getVectorDBEnvs = (
           name: "LLAMA_CLOUD_INDEX_NAME",
```

Comment:
how about setting these env variables in the spec.ts instead of using `ciInfo`? then we don't have any side effects in our code